providers/oidc.go

```diff
@@ -231,8 +239,10 @@ func getOIDCHeader(accessToken string) http.Header {
 }
 
 func (p *OIDCProvider) findClaimsFromIDToken(ctx context.Context, idToken *oidc.IDToken, accessToken string, profileURL string) (*OIDCClaims, error) {
+	claims := &OIDCClaims{
+		Groups: []string{},
```

Comment:
I don't think you need this, right?

Reply:
Think you are right works fine without, removed!